x-pack/platform/plugins/shared/cases/server/common/types/id_incrementer.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { SavedObject } from '@kbn/core-saved-objects-server';
+import { CaseIdIncrementerAttributesRt } from '../../../common/types/domain/incremental_id/latest';
+
+export interface CaseIdIncrementerPersistedAttributes {
+  '@timestamp': number;
+  last_id: number;
```

Comment:
I'm back and forth on the best intuitive name for this. Should it be `last_id` for the last id applied or should we be obvious like `next_id` or even `next_id_to_be_applied` or something. Last ID though does make the syncing with the last ID applied when we make that check easier... 🤔

Reply:
Not urgent enough to change, plus the saved object definitions were already merged here https://github.com/elastic/kibana/pull/219757. Thanks for the recommendations!